tbf-challenge/teamwork-be
# add likes count to get article endpoint 319

#### Pre-Submission Checklist






#### Checklist:





#### What does this PR do?


https://github.com/tbf-challenge/teamwork-be/issues/319
#### How should this be manually tested?


1. Start the server with npm 
2. Make a GET request to this endpoint http://localhost:3000/api/v1/articles/3
3. This should successfully return the sample data below
```
{
  "status": "success",
  "data": {
    "userId": 10,
    "articleId": 10,
    "createdOn": "2019-05-17",
    "articleTitle": "How to become a 10x software developer",
    "article": "/You are expected to create a set of ​ API endpoints",
    "published": true,
    "likesCount": 3,
    "comments": [
      {
        "commentId": 10,
        "comment": "This post articulates all there is to be a 10x software developer",
        "authorId": 10
      }
    ]
  }
}
```

##### Any background context you want to provide?



#### What are the relevant issues?
https://github.com/tbf-challenge/teamwork-be/issues/319
